crates/sui-core/src/authority/test_authority_builder.rs

```diff
@@ -118,19 +119,31 @@ impl<'a> TestAuthorityBuilder<'a> {
             std::fs::create_dir(&store_base_path).unwrap();
             store_base_path
         });
-        let authority_store = match self.store {
-            Some(store) => store,
+        let (authority_store, empty_database) = match self.store {
+            Some(store) => {
+                let empty_database = store
+                    .database_is_empty()
+                    .expect("Database read should not fail at init.");
+                (store, empty_database)
+            }
             None => {
+                let perpetual_tables =
+                    Arc::new(AuthorityPerpetualTables::open(&path.join("store"), None));
+                let empty_database = perpetual_tables
```

Comment:
I don't think we need this in test authority builder.
Test authority builder is used to create a single authority, which won't be able to run epoch-conservation check anyway.

Reply:
ok so removing the change for this file entirely